### PR TITLE
Link Instructors with multiple courses

### PR DESCRIPTION
**Pull request by burhandodhy**
**Description**
This PR fixes the issue where one instructor can't be linked with multiple courses.

image

**Jira ticket**
https://edlyio.atlassian.net/browse/EDLY-2719

### CODE REVIEW

File: course_discovery/apps/course_metadata/data_loaders/tests/test_api.py [hunk 1]
```diff
@@ -22,7 +22,8 @@
 from course_discovery.apps.course_metadata.data_loaders.tests import JPEG, JSON, mock_data
 from course_discovery.apps.course_metadata.data_loaders.tests.mixins import DataLoaderTestMixin
 from course_discovery.apps.course_metadata.models import (
-    Course, CourseEntitlement, CourseRun, CourseRunType, CourseType, Organization, Person, Program, ProgramType, Seat, SeatType
+    Course, CourseEntitlement, CourseRun, CourseRunType, CourseType, Organization, Person, Program, ProgramType, Seat,
+    SeatType
```
Comment (rehan99000) [inline course_discovery/apps/course_metadata/data_loaders/tests/test_api.py:26]: why this change?

**Review by iam-mhaseeb — APPROVED** [review]
👍

**Review by rehan99000 — APPROVED** [review]
a question

File: course_discovery/apps/course_metadata/data_loaders/tests/test_api.py (hunk 1 above)
Comment (burhandodhy) [reply course_discovery/apps/course_metadata/data_loaders/tests/test_api.py:26]: `make quality` recommend this change

File: course_discovery/apps/course_metadata/data_loaders/api.py [hunk 2]
```diff
@@ -19,8 +19,8 @@
 from course_discovery.apps.course_metadata.data_loaders import AbstractDataLoader
 from course_discovery.apps.course_metadata.data_loaders.course_type import calculate_course_type
 from course_discovery.apps.course_metadata.models import (
-    Course, CourseEntitlement, CourseRun, CourseRunType, CourseType, Organization, Program, ProgramType, Seat, SeatType,
-    Subject, Video, Person, PersonSocialNetwork
+    Course, CourseEntitlement, CourseRun, CourseRunType, CourseType, Organization, Person, PersonSocialNetwork, Program,
+    ProgramType, Seat, SeatType, Subject, Video
```
Comment (zubair-arbi) [inline course_discovery/apps/course_metadata/data_loaders/api.py:23]: Put every item in new line.

**Review by zubair-arbi — APPROVED** [review]
Just one suggestion otherwise LGTM 👍

File: course_discovery/apps/course_metadata/data_loaders/api.py (hunk 2 above)
Comment (burhandodhy) [reply course_discovery/apps/course_metadata/data_loaders/api.py:23]: Quality checks raise an error when every item added in a new line 

image
